test/fluree/db/policy/query_test.clj

```diff
@@ -293,3 +290,95 @@
         (is (= []
                @(fluree/query policy-deny other-query))
             "ex:Other class should be restricted")))))
+
+(deftest policy-values-test
+  (let [conn @(fluree/connect-memory)
+        db @(fluree/create-with-txn conn {"@context" [test-utils/default-str-context]
+                                          "ledger" "policy/values"
+                                          "insert"
+                                          [{"@id" "usa:wi"
+                                            "@type" "usa:state"
+                                            "ex:name" "Wisconsin"
+                                            "ex:capitol" "Madison"}
+                                           {"@id" "usa:ny"
+                                            "@type" "usa:state"
+                                            "ex:name" "New York"
+                                            "ex:capitol" "Albany"}
+                                           {"@id" "usa:nc"
+                                            "@type" "usa:state"
+                                            "ex:name" "North Carolina"
+                                            "ex:capitol" "Charlotte"}
+                                           {"@id" "usa:co"
+                                            "@type" "usa:state"
+                                            "ex:name" "Colorado"
+                                            "ex:capitol" "Denver"}
+                                           {"@id" "usa:pr"
+                                            "@type" "usa:territory"
+                                            "ex:name" "Puerto Rico"
+                                            "ex:capitol" "San Juan"}]})]
+    (testing "no policyValues"
```

Comment:
What's supposed to happen when there are no policyValues? This test needs more commentary